packages/ui/src/watch/home-page/settings/upload-button/index.tsx

```
@@ -48,6 +43,8 @@ const UploadButton = () => {
     );
   }
 
+  const enabled = data && data['upload'];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)